### ISSUE TITLE
FS25 Integration

### ISSUE THREAD

**Issue by Bulldogghp** [Fetty42/FS22_VIPOrderManager#17]
Good Afternoon Fetty42. First off I'd like to thank you for a fantastic mod, it has made FS22 even more enjoyable. Can you possible make VIP Order Manager available for FS25? 

Thank you for your time! I look forward to hearing from you.

**Comment by Fetty42** [comment]
Yes, I will try to provide the mod for LS25. Hopefully I'll have some time during the Christmas holidays.

**Comment by Bulldogghp** [comment]
Thank you for your response to my previous post. Do you by chance have an idea when this mod would be released to FS25? I appreciate your time, thanks!

**Comment by Fetty42** [comment]
Hi @Bulldogghp, I have already submitted the mod to ModHub and am also waiting for approval.

**Comment by Bulldogghp** [comment]
That is fantastic! Much appreciated!
